### ISSUE TITLE
Current release nominally incompatible with Sphinx >= 5

### ISSUE THREAD

**Issue by sleiner** [useblocks/sphinx-data-viewer#12]
The current released version of this package (1.2.2) requires Sphinx to be `<5,>4`. This means it would be incompatible with all Sphinx versions from 5.0 on. In my experience, it works just fine, but it is a problem when using strict dependency managers such as [PDM](https://pdm.fming.dev), which (as far as I can tell, rightfully) reports:

```
% pdm add sphinx-data-viewer
Adding packages to default dependencies: sphinx-data-viewer
🔒 Lock failed
Unable to find a resolution for sphinx
because of the following conflicts:
  sphinx<5,>=4 (from sphinx-data-viewer@0.1.0)
  sphinx<5,>=4 (from sphinx-data-viewer@0.1.1)
  sphinx<5,>=4 (from sphinx-data-viewer@0.1.2)
  sphinx>=7.0.1 (from project)
To fix this, you could loosen the dependency version constraints in pyproject.toml. See https://pdm.fming.dev/latest/usage/dependency/#solve-the-locking-failure for more details.
See /var/folders/vx/hn90mz9j7x1bw6j_msm723h80000gn/T/pdm-lock-lj7c7wo_.log for detailed debug log.
[ResolutionImpossible]: Unable to find a resolution
Add '-v' to see the detailed traceback
```

## Proposed fix
Since 720c7f3d66f2e47c4455a2cfe90cfdc3a1c8e50c, there is [no more upper bound on Sphinx](https://github.com/useblocks/sphinx-data-viewer/blob/720c7f3d66f2e47c4455a2cfe90cfdc3a1c8e50c/pyproject.toml#L13). So releasing the current state `main` would solve the issue :-)

**Comment by sigma67** [comment]
@useblocks any plans to publish a release?

**Comment by sigma67** [comment]
@danwos perhaps? :)

**Comment by danwos** [comment]
Sorry for the "short" delay. Mea culpa.

I just want to merge also PR #10, which adds jquery support for the newer Sphinx versions.

**Comment by sigma67** [comment]
It seems there is nothing happening on #10 .. would you mind creating a release to at least fix the issue mentioned in this thread?

**Comment by sigma67** [comment]
@danwos It's 2024 and this package is still on `sphinx<5`, can we please have a release? 🙏

**Comment by danwos** [comment]
Got released with `0.1.3`, sorry for the annoying delay.

**Comment by sigma67** [comment]
Amazing, thank you 🙏
